fr/basic/index.html

```diff
@@ -0,0 +1,995 @@
+      fichiers d'un coup.
+    </p>
+
+    <p class="nutshell">
+      <strong>Pour résumer</strong>, vous utiliserez <code>git add</code> pour
+      commencer à suivre de nouveaux fichiers et aussi pour mettre en attente
+      des modifications apportées à des fichiers déjà suivis, puis
+      <code>git status</code> et <code>git diff</code> pour voir ce qui a été
+      modifié et mis en attente et finalement <code>git commit</code> pour
+      enregistrer un instantané dans votre historique. Ce sera le flux de travail
+      basique que vous utiliserez la plupart du temps.
+    </p>
+
+  </div>
+</div>
+
+<div class="box">
+  <h2>
+    <span class="docs">
+      <a target="new" href="http://git-scm.com/docs/git-add">docs</a> &nbsp;
+      <a target="new" href="http://git-scm.com/book/fr/Les-bases-de-Git-Enregistrer-des-modifications-dans-le-d%C3%A9p%C3%B4t#Placer-de-nouveaux-fichiers-sous-suivi-de-version">livre</a>
+    </span>
+    <a name="add">git add</a>
+    <span class="desc">ajouter le contenu d'un fichier dans la zone d'attente</span>
+  </h2>
+
+  <div class="block">
+    <p>
+      Dans Git, vous devez ajouter le contenu d'un fichier dans la zone d'attente
+      avant de pouvoir le valider. Si c'est un nouveau fichier, vous pouvez
+      exécuter <code>git add</code> pour l'ajouter au préalable à la zone d'attente,
+      mais même si le fichier est déjà "suivi" - ie, il fait partie de votre dernière
+      validation - vous devez toujours lancer <code>git add</code> pour ajouter de
+      nouvelles modifications à votre zone d'attente. Voyons quelques exemples.
+    </p>
+
+    <p>Revenons à notre exemple Hello World, une fois initialisé notre projet,
+      nous commencerions à présent à y ajouter des fichiers et le ferions avec
+      <code>git add</code>. Nous pouvons utiliser <code>git status</code> pour
+      afficher l'état de notre projet.
+    </p>
+
+<pre>
+<b>$ git status -s</b>
+<span class="red">??</span> README
+<span class="red">??</span> hello.rb
+</pre>
+
+    Nous avons désormais deux fichiers non suivis. Nous pouvons maintenant les ajouter.
+
+<pre>
+<b>$ git add README hello.rb</b>
+</pre>
+
+    Maintenant si nous exécutons <code>git status</code> à nouveau, nous verrions
+    qu'ils ont été ajoutés.
+
+<pre>
+<b>$ git status -s</b>
+<span class="green">A</span>  README
+<span class="green">A</span>  hello.rb
+</pre>
+
+    <p class="aside">
+      Il est aussi courant d'ajouter récursivement tous les fichiers d'un nouveau projet
+      en indiquant le dossier de travail courant comme suit: <code>git add .</code>.
+      Sachant que Git va ajouter récursivement tous les fichiers dans un dossier que
+      vous lui donnez, si vous lui donnez le répertoire de travail courant, il va
+      simplement faire le suivi de chaque fichier qui s'y trouve. Dans ce cas,
+      la commande <code>git add .</code> aurait donné la même chose que
+      <code>git add README hello.rb</code>, et par la même également
```

Comment:
Pas "par là-même", qui a un tout autre sens que "for that matter", qu'on traduira plutôt par "d'ailleurs" ici.